demo-client/src/main/java/com/illia/client/model/request/entity/SortQueryEntity.java

```diff
@@ -0,0 +1,34 @@
+package com.illia.client.model.request.entity;
+
+import com.illia.client.model.request.registry.AttributeRegistry;
+import com.illia.client.model.request.registry.OperationsRegistry;
+import com.illia.client.model.request.registry.OrderRegistry;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import static com.illia.client.model.request.registry.OperationsRegistry.SORT;
+
+@EqualsAndHashCode(callSuper = false)
+@Data
+@Builder
+public class SortQueryEntity extends QueryEntity {
+    private AttributeRegistry attribute;
+    private OrderRegistry order;
+    private long limit;
+
+    @Override
+    public String getFileName() {
```

Comment:
you don't need to override them. Also, if you use lombok's Data annotation you don't even need any getters/setters for parent/child class